pkg/kubecli/kubecli.go

```diff
@@ -47,6 +47,10 @@ func Get() (*kubernetes.Clientset, error) {
 }
 
 func GetRESTClient() (*rest.RESTClient, error) {
+	return GetRESTClientFromFlags(master, kubeconfig)
```

Comment:
test suite injection. This is usually the preferred way of doing it over using globals.